File: app/src/main/java/kztproject/jp/splacounter/util/GameCountUtils.java
package kztproject.jp.splacounter.util;

import kztproject.jp.splacounter.model.Counter;

public class GameCountUtils {

    public static final int GAME_UNIT = 2;

    public static int convertGameCountFromCounter(Counter counter) {
        return counter.getCount() / GAME_UNIT;
    }

}

File: app/src/main/java/kztproject/jp/splacounter/util/GameCountUtils.kt
package kztproject.jp.splacounter.util

import kztproject.jp.splacounter.model.Counter

class GameCountUtils {
    companion object {
        const val GAME_UNIT = 2

        fun convertGameCountFromCounter(counter: Counter): Int {
            return counter.count / GAME_UNIT
        }
    }
}
